packages/cardano-services/test/entrypoints.test.ts

```diff
@@ -452,71 +452,6 @@ describe('entrypoints', () => {
       });
     });
 
-    describe('with unhealthy internal providers', () => {
```

Comment:
This is actually still a valid test. We don't want the service to start up if an internal provider is unhealthy

Reply:
How this could be determined since we remove provider health checks during service init? Should we consider then moving them in [startImpl](https://github.com/input-output-hk/cardano-js-sdk/blob/a2bfcc62c25e71d78d07b961267d7ce9679b6cf4/packages/cardano-services/src/Http/HttpServer.ts#L83-L86) for instance?

Reply:
Yes, unhealthy at the point of _starting_ is the logical place for this to fail. It will likely need a refactor of the `HttpService`, maybe to expose a new method for checking the health of internal providers.